src/cpp/session/resources/schema/user-prefs-schema.json

```diff
@@ -268,16 +268,18 @@
             "type": "boolean",
             "default": false,
             "title": "Continue comments after adding new line",
-            "description": "Whether continue comments (by inserting the comment character) after adding a new line."
+            "description": "Whether to continue comments (by inserting the comment character) after adding a new line.  Press Shift + Enter to exit a comment."
```

Comment:
Not sure why this is changed. Adding word "to" makes sense, but not sure why "Press Shift + Enter..." was added.